utilities/rpp-performancetests/HIP_NEW/CMakeLists.txt

```diff
@@ -1,17 +1,27 @@
 cmake_minimum_required(VERSION 3.0)
-project(Rpp_test)
+project(rpp_performancetests/HIP_NEW)
```

Comment:
@r-abishek Do we want to name this NEW? Similarly for the others? If the older ones are deprecated or removed maybe we can rename that and keep this as HIP?

Reply:
Yeah the NEW word has been in our pipeline to remove. It's present in the name of the folders too, but renaming the folders will cause git to interpret it as a re-creation of all files inside. Test suite renaming with some updates will be done as a separate PR